Package.swift

```diff
@@ -5,9 +5,8 @@ let package = Package(
     name: "Sentry",
     platforms: [.iOS(.v11), .macOS(.v10_13), .tvOS(.v11), .watchOS(.v4)],
     products: [
-        .library(name: "Sentry", targets: ["Sentry"]),
-        .library(name: "Sentry-Dynamic", type: .dynamic, targets: ["Sentry"]),
-        .library(name: "SentrySwiftUI", targets: ["SentrySwiftUI"])
+        .library(name: "SentryOnly", type: .dynamic, targets: ["Sentry"]),
+        .library(name: "SentryWithSwiftUI", type: .dynamic, targets: ["SentrySwiftUI"])
```

Comment:
h: This is a breaking change.
If we go forward with the type changing we should keep the names.

Reply:
Note that the PR description explains what has necessitated this.

Reply:
Can't we change the target names instead, @lhunath? I would not like to move forward with this change, as already pointed out by @brustolin this is a breaking change.

Reply:
@philipphofmann the target name is what defines how it will be imported in the project files, like `import Sentry`
This is also a breaking change. 

What I did not get it yet is why we cant keep the same name for library and target? 
I could not look into this yet.

Reply:
It's related to the fact that the `SentrySwiftUI` target has a dependency on `"Sentry"`, and without disambiguating the product and target names, SPM errors out. Maybe Firebase's approach of declaring a dependency on an intermediary wrapper target will work. I'm not sufficiently familiar with SPM to tell.